src/scss/tooltip.scss

```
@@ -0,0 +1,197 @@
+@import "o-overlay/main";
+@import "o-visual-effects/main";
+
+@import "src/scss/variables";
+
+/// Base tooltip style. Inherits from oOverlay.
+@mixin oTooltip {
+	@include oOverlay();
+
+	// Override border until it's updated in o-overlay
+	border-radius: $o-normalise-border-radius;
+	border: 0;
+
+	@include oEffectsShadowsElevation($elevation: 'high');
```

`@includes` should be at the top of the mixin - it should throw a verify error.